### PR TITLE
[ZEPPELIN-5481] Support to run R interpreter in yarn mode with customized conda env

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Simple PR to make R interpreter run in yarn mode with customized conda env. 

### What type of PR is it?
[ Feature ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5481

### How should this be tested?
* Manually tested

### Screenshots (if appropriate)
image

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: rlang/src/main/java/org/apache/zeppelin/r/IRInterpreter.java [hunk 1]
```diff
@@ -152,6 +153,17 @@ protected void initIRKernel() throws IOException, InterpreterException {
     }
   }
 
+  @Override
+  protected Map<String, String> setupKernelEnv() throws IOException {
+    Map<String, String> envs = super.setupKernelEnv();
+    String pathEnv = envs.getOrDefault("PATH", "");
+    if (condaEnv != null) {
+      pathEnv = new File(".").getAbsolutePath() + "/" + condaEnv + "/bin:" + pathEnv;
```
Comment (Reamer) [inline rlang/src/main/java/org/apache/zeppelin/r/IRInterpreter.java:161]: Please use `File.separator` and `File.pathSeparator` to build the `PATH` environment variable.

**Review by Reamer — CHANGES_REQUESTED** [review]

File: rlang/src/main/java/org/apache/zeppelin/r/IRInterpreter.java (hunk 1 above)
Comment (zjffdu) [reply rlang/src/main/java/org/apache/zeppelin/r/IRInterpreter.java]: Fixed

Comment (Reamer) [reply rlang/src/main/java/org/apache/zeppelin/r/IRInterpreter.java]: Looks weirder than I thought, but I think that should be correct.

**Review by Reamer — APPROVED** [review]
LGTM

**Comment by zjffdu** [comment]
Will merge if no more comment

**Pull request closed without merging**